internal/catalog/internal/controllers/workloadhealth/controller_test.go

```diff
@@ -562,7 +567,13 @@ func (suite *workloadHealthControllerTestSuite) waitForReconciliation(id *pbreso
 }
 
 func TestWorkloadHealthController(t *testing.T) {
-	suite.Run(t, new(workloadHealthControllerTestSuite))
+	for _, tenancy := range resourcetest.TestTenancies() {
```

Comment:
My two cents on this approach

1. Not all test cases within a suite will need to test tenancy
2. Not all parts of a test case need to be subjected to the tenancy loop. For example, consider an integration test case that starts the controller manager and tests reconciliation. Only the reconcile logic needs to be subjected to tenancy based checks.

Reply:
ok got it.